README.md

```diff
@@ -18,6 +18,12 @@ This information reflects the head of this branch.
 | Compatible with CSI Version                                                                | Container Image                                         | [Min K8s Version](https://kubernetes-csi.github.io/docs/kubernetes-compatibility.html#minimum-version) |
 | ------------------------------------------------------------------------------------------ | ------------------------------------------------------- | --------------- |
 | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | k8s.gcr.io/sig-storage/csi-node-driver-registrar        | 1.13            |
+| ------------------------------------------------------------------------------------------ | ------------------------------------------------------- | --------------- |
+| [CSI Spec v1.1.0](https://github.com/container-storage-interface/spec/releases/tag/v1.1.0) | k8s.gcr.io/sig-storage/csi-node-driver-registrar        | 1.13            |
+| ------------------------------------------------------------------------------------------ | ------------------------------------------------------- | --------------- |
+| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | k8s.gcr.io/sig-storage/csi-node-driver-registrar        | 1.13            |
+| ------------------------------------------------------------------------------------------ | ------------------------------------------------------- | --------------- |
+| [CSI Spec v1.3.0](https://github.com/container-storage-interface/spec/releases/tag/v1.3.0) | k8s.gcr.io/sig-storage/csi-node-driver-registrar        | 1.13            |
```

Comment:
I would remove all the other lines and just keep the 1.3.0 line. The CSI spec is supposed to be backwards compatible to 1.0

Reply:
sure. @msau42 addressed this in the patchset. PTAL .